packages/core/src/components/link/link.scss

```
@@ -9,11 +9,29 @@
     text-decoration: underline;
   }
 
+  ::slotted(*:hover) {
+    color: var(--tds-link-hover);
+    text-decoration: none;
+  }
+
+  .no-underline {
+    ::slotted(*) {
+      text-decoration: none;
+    }
+  }
+
+  .no-underline {
+    ::slotted(*:hover) {
+      color: var(--tds-link-hover);
+      text-decoration: underline;
+      text-decoration-color: var(--tds-link-hover);
+    }
+  }
```

Instead of this, you could just:

```suggestion
  .no-underline {
    ::slotted(*) {
      text-decoration: none;
    }
    ::slotted(*:hover) {
      color: var(--tds-link-hover);
      text-decoration: underline;
      text-decoration-color: var(--tds-link-hover);
    }
  }
```